### DISCUSSION TITLE
The Wait for 2.0 is Killing Me

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#394] [General]
I know we all need to wait for this update so it can be amazing, but waiting for it is a pain in the neck. I wish we had some estimate on when it could come out, or at least a teaser of what features will be added.

**Comment by ota-works** [comment]
Same, tbh. I wish Keiwando does a livestream and interacts with us.

I would also be cool if he could partner up with some pro animators who use the app and have an interaction with them as they use the app in a livestream, like Toon Boom Youtube channel does nowadays

**Comment by ota-works** [comment]
I keep refreshing the page here everyday to see if there's something new here. 
Needless to say, Keiwan has built up the hype unintentionally or intentionally.

**Comment by DWrightAnimation** [comment]
https://github.com/keiwando/toonsquid/assets/129681998/c80b7905-d40c-42dc-b67c-cf7cc396910c

I know Z-Axis nudging is something we all want in ToonSquid, to make the limbs go back and forth, but I managed to find a workaround. 

If I could only have 3 new features in ToonSquid right now, they'd be "Out of Pegs", "Animatable Blur and Other Effects" and "Z-Axis Nudging".

**Reply by ota-works** [reply]
I'm personally looking forward to more brushes and customization, Hide Layers and Non-Destructive Effects eg Blur, Glow, etc. like in old Flash. (Interface customization wouldn't be bad either)

**Reply by ota-works** [reply]
Oh yeah, and the vector trimming tool

**Comment by DinoAwakening** [comment]
https://github.com/keiwando/toonsquid/assets/120270913/76db1d43-f195-465a-ac43-e929d039944a

Let’s just assume for a moment that the version is waiting to be released next is 2.0. Hope Keiwando don’t get to make the sequence like 1.10 or something cause it’ll mean less features per update :’D

**Comment by ota-works** [comment]
This isn't the place to ask and it can be deleted when answered, but how does one import Toonsquid brushes into Toonsquid? Is it the regular click-and-drag or is there a specific way of doing it?

**Reply by DWrightAnimation** [reply]
Just click import on the opening screen for ToonSquid, then it'll open a window for the Files app, just find your brush you want to import, then click on it. It'll look like nothing happened at first, but once you open a project and look through the brushes, you'll find a folder that has the name of your brush on it.

**Reply by ota-works** [reply]
Thanks! I'll give it a shot

**Reply by ota-works** [reply]
Many thanks! It works!!!

**Comment by DinoAwakening** [comment]
NOOOOOOOOOOOOO!

I got excited for a moment, and all we got was a Bug Fixes version :(

**Reply by ota-works** [reply]
Ouch! I just saw the update.

I wish he would announce the date to look forward to the major updates, like Procreate, but Keiwan is only 1 person working on this, so we shouldn't pressure him so much. I wish he got a full-time team to work on this. I'm pretty sure some people here are willing to sign up.

**Reply by DWrightAnimation** [reply]
Well, based on previous updates, we should have 1 to 4 more bug fix updates before 2.0, which means it could come out by Christmas.

**Reply by DinoAwakening** [reply]
I hope it does. That way, Keiwando would be having almost half a year (since 1.9.0) to development and polish new features and tweaks to be considered a MASSIVE well done and revolutionary update for ToonSquid :)

**Reply by DWrightAnimation** [reply]
Well, now ToonSquid is on 1.9.6, which could either mean 2.0 is next, or we have 3 more smaller updates before 2.0. Hoping Out of Pegs is in 2.0.
